etc/systemd/system/Makefile.am

```diff
@@ -8,6 +8,7 @@ systemdunit_DATA = \
 	zfs-mount.service \
 	zfs-share.service \
 	zfs-import.target \
+	zfs-volume.service \
```

Comment:
nit: since there already aren't alphabetic I'd move this up below the other .service files